### PR TITLE
Issue#101 add perpetrators part in update report part

### PR DESCRIPTION
**Pull request by eunyyyoung**
Issue #101 
Issue #99

### CODE REVIEW

**Comment by eunyyyoung** [comment]
Commit message #100 is a typing error

File: Backend/router/v1/report.ctrl.js [hunk 1]
```diff
@@ -28,8 +28,10 @@ const createReport = (req, res) => {
       facebook_url: req.body.data[0].facebook_url
     }
   }).then((result) => {
-    let pid = result[0].id;
-
+    let pid = null;
```
[the remark this answers is not in the document]
Comment (mtnviewlab) [reply Backend/router/v1/report.ctrl.js]: pid usually stands for process ID. :)

[the remark this answers is not in the document]
Comment (shberry) [reply Backend/router/v1/report.ctrl.js]: Yup :) No wonder pid sounded familiar but the context didn't feel right here haha
(In the commerce world, pid stands for product_id 😄 )

[the remark this answers is not in the document]
Comment (eunyyyoung) [reply Backend/router/v1/report.ctrl.js]: Oh I got it!

**Pull request merged**